src/org/thoughtcrime/securesms/ConversationFragment.java

```diff
@@ -660,6 +706,18 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
           handleSaveAttachment((MediaMmsMessageRecord)getSelectedMessageRecord());
           actionMode.finish();
           return true;
+        case R.id.menu_context_pin_message:
```

Comment:
Weird spacing here, does it show up like this locally?

Reply:
It looks better locally, my tab spacing was set up weird. I will attempt to fix this.